src/main/java/hillel/spring/HomeWork3/Hw3Repo.java

```diff
@@ -0,0 +1,14 @@
+package hillel.spring.HomeWork3;
+
+
+import lombok.Data;
+import org.springframework.stereotype.Repository;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Data
+@Repository
+public class Hw3Repo {
```

Comment:
Repository  is not a `@Data` class. Yes, in this implementation it looks like it is, but it is not.
Just use `@AllArgsConstructor` and provide a method for retrieving the data.